portal-api/src/v1/routes.js

```diff
@@ -103,9 +103,10 @@ authRouter
   .get(mandatoryCriteria.findOne)
   .delete(validateUserHasAtLeastOneAllowedRole({ allowedRoles: [ADMIN] }), mandatoryCriteria.delete);
 
-authRouter.route('/users').get(users.list).post(users.create);
-authRouter.route('/users/:_id').get(users.findById).put(users.updateById).delete(users.remove);
-authRouter.route('/users/:_id/disable').delete(users.disable);
+authRouter.route('/users').get(users.list).post(validateUserHasAtLeastOneAllowedRole({ allowedRoles: [ADMIN] }), users.create);
+authRouter.route('/users/:_id').get(validateUserIsActingOnSelfToChangePasswordOrIsAdmin, users.list);
+authRouter.route('/users/:_id').get(users.findById).put(users.updateById).delete(validateUserHasAtLeastOneAllowedRole({ allowedRoles: [ADMIN] }), users.remove);
```

Comment:
Is `disabled` endpoint deprecated?

Reply:
If yes then please update the swagger documentation and mark this PR as breaking.

Reply:
According to point 9 https://ukef-dtfs.atlassian.net/browse/DTFS2-6655 `disabled` endpoint is no longer used. Marked PR as breaking

Reply:
Has the swagger documentation been updated or does it not exists? Thanks

Reply:
No, I don't think it exists.